File: CILantroTools/cilantro-tools-web-ui/src/api/models/tests/input-output/elements/IntElement.ts
interface IntElement {
	type: 'Byte' | 'Short' | 'Int' | 'Long' | 'Sbyte' | 'Uint' | 'Ulong' | 'Ushort';
	name: string;
	minValue: number;
	maxValue: number;
	description?: string;
	excludeZero?: boolean;
}

export default IntElement;

File: CILantroTools/CILantroToolsWebAPI/CILantroToolsWebAPI/Models/Tests/InputOutput/Elements/IntElement.cs
namespace CILantroToolsWebAPI.Models.Tests.InputOutput.Elements
{
    public class IntElement : AbstractInputOutputElement
    {
        public string Name { get; set; }

        public int MinValue { get; set; }

        public int MaxValue { get; set; }

        public bool ExcludeZero { get; set; }
    }
}